hw/ip/otbn/dv/otbnsim/sim/wsr.py

```diff
@@ -149,17 +149,90 @@ def set_unsigned(self, value: int) -> None:
         self._random_value = value
 
 
+class URNDWSR(WSR):
+    '''Models URND PRNG Structure'''
+    def __init__(self, name: str):
+        super().__init__(name)
+        self.traced = False
+        self._seed = [0x84ddfadaf7e1134d, 0x70aa1c59de6197ff,
+                      0x25a4fe335d095f1e, 0x2cba89acbe4a07e9]
+        self.state_256b = [self._seed,
+                           4 * [0], 4 * [0],
+                           4 * [0], 4 * [0]]
+        self.URND = 4 * [0]
+        self._next_value = 0
```

Comment:
Could we change this so `_next_value` is `None` unless we've run `step`? It would match things like the PC calculation a bit better and avoids zero being a magic number. (Then you'd want to condition the update in `commit` on `_next_value is not None`)